src/cljc/centrebull/spec.cljc

```diff
@@ -74,7 +74,7 @@
 ;Clojure spec predicate for a non empty string
 (def non-empty-string (s/and string? #(not= "" %)))
 
-(defn- str->int [s] #?(:cljs (cond (and (string? s) #(not= "" %)) (cljs.tools.reader/read-string s)
+(defn- str->int [s] #?(:cljs (cond (and (string? s) #(not= "" %)) s
```

Comment:
I was getting `Uncaught TypeError: Cannot read property 'reader' of undefined`. I don't completely understand what this function does but it works with this change